### PR TITLE
fix(v2): add link to form guide in form builder

### LINKED ISSUE #4478: [0.5] Link to form guide in form builder
Current form guide button is not very discoverable as it is on the dashboard rather than the builder, even though users are likely to encounter more issues on the builder.

### PR DESCRIPTION
**Pull request by wanlingt**
## Solution

This PR moves the link to the form guide from the admin navbar to the form builder dashboard, so that users can discover it more easily.

Also added a new `subtle` colour theme for the icon button - [figma here](https://www.figma.com/file/u00dlQLsaQfvqv6ot5ZrJZ/Form-Design-System?node-id=2153%3A24)

Closes #4478 

**Breaking Changes** 

- [x] No - this PR is backwards compatible  

## Before & After Screenshots

**AFTER**:
image

## Tests

- [x] Click the help button in the form builder, it should open up to the form user guide

### CODE REVIEW

**Review by timotheeg — APPROVED** [review]
LGTM. Took me a while to find the `?` in the screenshot, LOL 😅 

BTW, what is the magic that pushes the icon all the way to the bottom? Does the `Stack` you used to wrap the other icons extend to take all available height?

**Comment by wanlingt** [comment]
> BTW, what is the magic that pushes the icon all the way to the bottom? Does the `Stack` you used to wrap the other icons extend to take all available height?

it is indeed magic

**Comment by wanlingt** [comment]
> BTW, what is the magic that pushes the icon all the way to the bottom? Does the `Stack` you used to wrap the other icons extend to take all available height?

jk yes it's the [parent `Stack`](https://github.com/opengovsg/FormSG/pull/4960/files#diff-ddce4a1d79dab1565ee3c4ca8cfe0ae34b442bb9cf7b043dec385a6c2756619eR54) which wraps the question mark and the other icons, and also provides the white background :)

**Comment by timotheeg** [comment]
Ah, I see it! `justifyContent="space-between"`

File: frontend/src/theme/components/Button.ts [hunk 1]
```diff
@@ -73,12 +81,13 @@ const genVariantOutlineColours = ({
...
   const { colorScheme: c } = props
-  const { bg, hoverBg, activeBg, focusBoxShadow } = genVariantSolidColours(c)
+  const { bg, hoverBg, activeBg, focusBoxShadow, color } =
+    genVariantSolidColours(c)
 
   return {
     bg,
     borderColor: bg,
-    color: 'white',
+    color: color,
```
Comment (timotheeg) [inline frontend/src/theme/components/Button.ts]: /nit

```suggestion
    color,
```

**Review by timotheeg — APPROVED** [review]

File: frontend/src/theme/components/Button.ts (hunk 1 above)
Comment (wanlingt) [reply frontend/src/theme/components/Button.ts]: [4fb3805](https://github.com/opengovsg/FormSG/pull/4960/commits/4fb380589ff1c8daa8c6257e1e2c6f9e828baa67)

**Comment by karrui** [comment]
should we update button stories for the new button type too?

**Comment by wanlingt** [comment]
> should we update button stories for the new button type too?

done here! [273c40c](https://github.com/opengovsg/FormSG/pull/4960/commits/273c40cc116affca528460b91285badb2a9ef5be)

**Review by timotheeg — APPROVED** [review]

**Comment by wanlingt** [comment]
now just waiting for design review :)

File: frontend/src/features/admin-form/create/common/CreatePageSidebar/CreatePageSidebar.tsx [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import { useCallback } from 'react'
-import { BiGitMerge } from 'react-icons/bi'
-import { Stack } from '@chakra-ui/react'
+import { BiGitMerge, BiQuestionMark } from 'react-icons/bi'
+import { Stack, Tooltip } from '@chakra-ui/react'
```
Comment (justynoh) [inline frontend/src/features/admin-form/create/common/CreatePageSidebar/CreatePageSidebar.tsx]:
```suggestion
import { Stack } from '@chakra-ui/react'
import Tooltip from '~components/Tooltip'
```

Comment (wanlingt) [reply frontend/src/features/admin-form/create/common/CreatePageSidebar/CreatePageSidebar.tsx]: thank you!! [24ab287](https://github.com/opengovsg/FormSG/pull/4960/commits/24ab2876e7c21e1dc239b6a2b3d2aebc012eb919)

**Pull request merged**